src/implementations/twig/components/tag/tag.story.js

```diff
@@ -0,0 +1,67 @@
+import { withKnobs, text, select, optionsKnob } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl/storybook-addon-notes';
+import withCode from '@ecl/storybook-addon-code';
+import { getExtraKnobs, tabLabels, getComplianceKnob } from '@ecl/story-utils';
+
+import defaultSprite from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
+import dataLink from '@ecl/specs-component-tag/demo/data--link';
+import dataRemovable from '@ecl/specs-component-tag/demo/data--removable';
+import dataDisplay from '@ecl/specs-component-tag/demo/data--display';
+
+import tag from './tag.html.twig';
+import notes from './README.md';
+
+// Preserve the adapted specs.
+const prepareTag = (data, link, aria) => {
+  data.tag.type = select(
+    'tag.type',
+    [data.tag.type],
+    data.tag.type,
+    tabLabels.required
+  );
+  data.tag.label = text('tag.label', data.tag.label, tabLabels.required);
+  if (link) {
+    data.tag.path = text('tag.path', data.tag.path, tabLabels.required);
+  }
+  if (aria) {
+    data.tag.aria_label = text(
+      'tag.aria_label',
+      data.tag.aria_label,
+      tabLabels.required
+    );
+  }
+  if (aria) {
+    data.default_icon_path = optionsKnob(
+      'default_icon_path',
+      { current: defaultSprite, 'no path': '' },
+      defaultSprite,
+      { display: 'inline-radio' },
+      tabLabels.required
+    );
+  }
+
+  getExtraKnobs(data);
+  getComplianceKnob(data);
+
+  return data;
+};
+
+export default {
+  title: 'Components/Tag',
+  decorators: [withKnobs, withNotes, withCode],
+};
+
+export const Display = () => tag(prepareTag(dataDisplay, false, true));
```

Comment:
```suggestion
export const Display = () => tag(prepareTag(dataDisplay));
```
We have this small bug in ecl-twig, we expose the aria_label prop for the display type in a knob, but the template only applies that to the removable.